doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -332,6 +332,9 @@ Modem libraries
   * Added:
 
     * Support for the application to send the Wi-Fi access point list to the cloud.
+    * Introduced :kconfig:option:`CONFIG_LOCATION_SERVICE_EXTERNAL`, which replaces the following configurations that are removed:
+      ``CONFIG_LOCATION_METHOD_GNSS_AGPS_EXTERNAL``, ``CONFIG_LOCATION_METHOD_GNSS_PGPS_EXTERNAL`` and ``CONFIG_LOCATION_METHOD_CELLULAR_EXTERNAL``.
+      New configuration handles Wi-Fi positioning also.
```

Comment:
```suggestion
      The new configuration handles also Wi-Fi positioning.
```

Reply:
I'm trying how this should be so that it's aligned correctly and not in the last bullet of the previous list. I don't know if empty line keeps this intended with correct list. We'll see soon.

Reply:
Yep, an empty line would be needed after the last bullet. Sorry for sloppy suggestions.